### PR TITLE
refactor: remove security executor deployment

### PR DESCRIPTION
**Pull request by florian-hoenicke**
the other executors inherit from the security executor, so it is not needed alone

### CODE REVIEW

**Comment by makram93** [comment]
Why are we removing security exectuor?

File: now/deployment/flow.py [hunk 1]
```diff
@@ -104,44 +104,15 @@ def deploy_k8s(f, ns, tmpdir, kubectl_path):
...
-
-
 @time_profiler
 def deploy_flow(
     deployment_type: str,
     flow_yaml: str,
     ns: str,
     env_dict: Dict,
     kubectl_path: str,
-    secured: bool = False,
```
Comment (makram93) [inline now/deployment/flow.py:137]: Where or how are these informations passed on to the flow?

Comment (florian-hoenicke) [reply now/deployment/flow.py:137]: good observation. I think that could be the reason why the local test is not passing. I will investigate

Comment (florian-hoenicke) [reply now/deployment/flow.py:137]: In case, there are no admin emails, user emails and api_keys, the flow knows it is not secured

**Pull request merged**